include/mercury/kw/kw.h

```diff
@@ -33,7 +33,9 @@ class KanoWorld
      *
      * \param client    (Optional) The HTTP client to use for requests
      */
-    explicit KanoWorld(std::shared_ptr<IHTTPClient> client =
+    explicit KanoWorld(
+            const std::string& url = "https://worldapi.kano.me",
```

Comment:
At the top of the file there is `using std::string;` yet there's a few places where `std::string` and `string` is used. Recommend picking one or the other and being consistent.

Reply:
Ditched `std::` in most places and added more items to the `using::...` section.